app/controllers/application_controller.rb

```diff
@@ -14,4 +16,12 @@ def authenticate_admin!
       redirect_to request.referrer || root_path, notice: "Admins Only! :|"
     end
   end
+
+  def send_slack_notification msg
+    uri = URI('https://slack.com/api/chat.postMessage')
+    params = { :token => Rails.application.secrets.slack_token, :channel => Rails.application.secrets.slack_channel, :text => msg, :pretty => 1 }
+    uri.query = URI.encode_www_form(params)
+
+    Net::HTTP.get_response(uri)
```

Comment:
There's a [slack ruby client](https://github.com/slack-ruby/slack-ruby-client/) you can consider using, instead of making the calls directly.